strings/round_start_sounds.txt

```diff
@@ -1,4 +1,7 @@
 sound/ambience/title1.ogg
 sound/ambience/title2.ogg
-sound/ambience/title3.ogg
 sound/ambience/clown.ogg
+config/title_music/bepoprush.ogg
```

Comment:
Not a fan of dontstopme. I'm actually really opposed to lyricals as lobby songs.

Bepoprush doesn't feel like it fits. There are better options.

Reply:
https://www.youtube.com/watch?v=YugU8hLahD8
https://www.youtube.com/watch?v=iN0fwj3vHfU
https://www.youtube.com/watch?v=wNPyHJl9Qs0
https://www.youtube.com/watch?v=m0SatLGwr0w

All of these fit about twenty times better.

Reply:
Oh here's another good one. https://www.youtube.com/watch?v=4pZeePlM3Jo

Reply:
I think this is going to be pretty subjective until we get some actual guidelines.

Reply:
I'm extremely opposed to dontstopme. I'm willing to tolerate bepop because it's not 'bad' but it doesn't fit. However dontstopme is just flat out jarring entirely against the intended mood in every way on top of being awful vocals.

Reply:
just just... tgs... just... database... ugh.